app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
+from app.players.player import Player
+
+
+def calculate_team_total_rating(players: list[Player]) -> int:
+    return sum(
+        [player.get_rating() for player in players]
+    )
```

Comment:
```suggestion
    return sum(player.get_rating() for player in players)
```
no need to create list here